apps/dapp/src/providers/SwapProvider.tsx

```diff
@@ -117,11 +125,19 @@ export const SwapProvider = (props: PropsWithChildren<{}>) => {
 
   useEffect(() => {
     const onMount = async () => {
-      try {
-        await sor.fetchPools();
-      } catch (e) {
-        console.log('failed to fetch sor pools');
-      }
+      return backOff(async () => {
+        const success = await balancerSwaps.fetchPools({
+          chainId: 1,
```

Comment:
What if we used the envChainMapping here, so the chainId isn't hard-coded?

Though I guess since the `templeDaiBalancerPool` isn't available on testnet it doesn't matter? Wdyt?